include/onnxruntime/core/framework/execution_provider.h

```diff
@@ -202,21 +202,23 @@ class IExecutionProvider {
 
   /**
      Indicate whether the graph capturing mode (e.g., cuda graph) is enabled for
-     the provider. Currently only CUDA execution provider supports it.
+     the provider. Currently only CUDA/TensorRT/Rocm execution providers support it.
```

Comment:
Remove `Currently only CUDA/TensorRT/Rocm execution providers support it`. 
This is interface so it shall not know which derived class implemented it.